metrics-httpclient/src/main/java/com/codahale/metrics/httpclient/InstrumentedHttpClientConnectionManager.java

```diff
@@ -30,80 +30,48 @@
 
     protected static Registry<ConnectionSocketFactory> getDefaultRegistry() {
         return RegistryBuilder.<ConnectionSocketFactory>create()
-            .register("http", PlainConnectionSocketFactory.getSocketFactory())
-            .register("https", SSLConnectionSocketFactory.getSocketFactory())
-            .build();
+                .register("http", PlainConnectionSocketFactory.getSocketFactory())
```

Comment:
Please don't add unrelated whitespace changes to your PR. This makes porting commits across different versions harder, because it increases the change of a possible merge conflict.

Reply:
ok, will undo